x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/querydsl/query/SpatialRelatesQuery.java

```diff
@@ -157,6 +166,11 @@ public ShapeRelation relation() {
         public Geometry shape() {
             return shape;
         }
+
+        @Override
+        public String toString() {
+            return Strings.toString(this, true, true);
```

Comment:
Yes. I did add something similar in some other work a few months back, but removed it to keep that PR clean. This time I wanted to keep it, because those exceptions were just too irritating.